app/views/task/_footer_links.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= button_to "To Do List", "/", method: :get %>
```

Comment:
I'm wondering why these footer links show up above the task lists on the root page